BParkingNano/plugins/TrackMerger.cc

```diff
@@ -154,11 +154,7 @@ void TrackMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup const
 
     // high purity requirment applied only in packedCands
     if( iTrk < nTracks && !trk.trackHighPurity()) continue;
-   
-    // build transient track
-    const reco::TransientTrack trackTT((*(trk.bestTrack())), &(*bFieldHandle));
-    if (!trackTT.isValid()) continue;
-   
+    const reco::TransientTrack trackTT(trk.pseudoTrack(), &(*bFieldHandle));
```

Comment:
what is the reason to change from bestTrack to pseudoTrack?

Reply:
Dear Sara,

Thanks for the review and sorry for the delay. No there is not particular reason. I was tesing if the results between the two functions and forgot to revert it. It seems that indeed they are similar. The reason for that test is that in the twiki the proposed function to use for reco::Track accessing is pseudotrack. I have no strong preference and I can revert it in no-time. Thanks again.

Best,
George

Reply:
hi George, ok, I don't know what is the best option.
can you share the link to the twiki you are referring to?

Reply:
Hi Sara,

Yes gladly, the twiki is here: https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookMiniAOD2017#Embedded_track_information 
the line in question is here:
  " the reco::Track of the candidate is provided by the pseudoTrack() method"

Best, george